host_applications/linux/apps/raspicam/RaspiStill.c

```diff
@@ -47,7 +47,9 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
 // We use some GNU extensions (asprintf, basename)
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
```

Comment:
Going to be a conflict here against #503. Please merge 503 first as it has the nicer indentation.